modules/tide_ckeditor/tide_ckeditor.install

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Install file for tide_ckeditor.
+ */
+
+use Drupal\user\Entity\Role;
+
+/**
+ * Install the tide_ckeditor module.
+ */
+function tide_ckeditor_install() {
+  tide_ckeditor_update_9001();
+}
+
+/**
+ * Assign `use text format advanced` to editor,approver and site_admin.
+ */
+function tide_ckeditor_update_9001() {
+  $role_ids = ['editor', 'approver', 'site_admin'];
+  foreach ($role_ids as $role_id) {
+    $role = Role::load($role_id);
+    $role->grantPermission('use text format advanced')->save();
+  }
```

Comment:
@vincent-gao forgot to mention, does the contributor role need this permission ? I think it should have this.